ospf6d/ospf6_nssa.c

```diff
@@ -1021,9 +1020,8 @@ static void ospf6_nssa_flush_area(struct ospf6_area *area)
 
 		/* Flush the translated LSA */
 		if (ospf6_check_and_set_router_abr(ospf6)) {
-			type = htons(OSPF6_LSTYPE_AS_EXTERNAL);
 			type5 = ospf6_lsdb_lookup(
-				htons(type), lsa->external_lsa_id,
+				htons(OSPF6_LSTYPE_AS_EXTERNAL), lsa->external_lsa_id,
```

Comment:
There are a couple of _missing_ `htons` in this file on lines 578 and 667. Could you fix them as well, please?

Reply:
And also fix the style warnings, please.

Reply:
@idryzhov style warnings and `htons` in line 578 were fixed. The one in 667 was already updated in a different PR #10158 that is still under review....